examples/api/amp-access/amp-access.js

```diff
@@ -66,7 +66,7 @@ function handleAuthorization(request, response) {
 
 function handleLogin(request, response) {
   const returnUrl = request.query.return;
-  const filePath = utils.project.absolute('/examples/static/samples/files/login.html');
+  const filePath = utils.project.absolute('/examples/api/amp-access/login.html');
```

Comment:
you can use relative paths instead `./login.html`.

Reply:
Do you mean by using ```utils.project.relative(...``` or just the string ```./login.html```?

Reply:
```./login.html``` only works, if you set up ```nunjucks.configure(__dirname)``` beforehand.
Was this what you were thinking of?
Either way, I will move the ```filePath``` into a constant.

Reply:
ah ok, `path.join(__dirname, 'login.html')` would've been my next suggestion.

Reply:
works for me, and 1 less line of code 👍